### ISSUE TITLE
Flutter client support 

### ISSUE THREAD

**Issue by gedw99** [SigNoz/signoz#179]
## Is your feature request related to a problem?

I think that it’s possible to provide dart support so that flutter web, mobile and desktop clients can be supported.

## Describe the solution you'd like

A repo that demonstrates flutter client using the system.

## Additional context

As I think about it open telemetry provides protobufs and flutter has good support for Protocol Buffers as well as GRPC and GRPC Web.

I have not had a chance to look at the golang server yet to see if GRPC or Protocol Buffers are being used however and just wanted to firstly raise the issue.

I have experience with flutter / golang GRPC interoperability and can help in this effort.

**Comment by pranay01** [comment]
Hey @gedw99, just to confirm - when you say flutter, you mean this - https://flutter.dev/ , right?

**Comment by gedw99** [comment]
> Hey @gedw99, just to confirm - when you say flutter, you mean - https://flutter.dev/, right?

Yes that’s correct.

**Comment by pranay01** [comment]
ok. I am not very familiar with flutter. Can you explain a bit on how does flutter interact with Golang/GRPC?

As I understand flutter is a framework for building mobile apps and web app (primarily the frontend) 

Do you mean adding flutter support in SigNoz will enable monitoring these mobile apps/ web apps built on flutter? 
Feel free to correct, if I am not following this correctly

**Comment by gedw99** [comment]
Yes that is what I mean. By adding support flutter apps get open telemetry and because flutter has good GRPC support is should be relatively straight forward .

i have done a Little prior art support but not found anyone else biting it off yet to do it

most flutter devs use sentry or firebase currently I think

**Comment by daya** [comment]
It will be a great addition and for me the only reason why I am not dumping Sentry which BTW is a  behemoth & painful to run self hosted primarily due to lack of documentation.

**Comment by alexanderadam** [comment]
> the only reason why I am not dumping Sentry which BTW is a behemoth & painful to run self hosted primarily due to lack of documentation.

Just use [GlitchTip](https://glitchtip.com/documentation/install/) then. It can be selfhosted for free, is compatible to Sentry but uses less resources and even supports [OAuth](https://glitchtip.com/documentation/install) and other features that SigNoz lack.

It's compatible to the Sentry API so you can keep the client libraries.

You're welcome. :wink:
